site/docs/configuration/expected-outputs/guardrail.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 101
+sidebar_label: Guardrail
+---
+
+# Guardrail
+
+Use the `guardrail` assert type to ensure that LLM outputs pass safety checks based on the provider's built-in guardrails.
```

Comment:
super nit: do we want this assert type to be `guardrails` instead of `guardrail` 🤔